theme_enark/static/src/js/tour.js

```diff
@@ -4,39 +4,44 @@ import * as wTourUtils from "@website/js/tours/tour_utils";
 
 const snippets = [
     {
-        id: 's_banner',
-        name: 'Banner',
-        groupName: "Intro",
+        id: 's_freegrid',
+        name: 'Free grid',
+        groupName: "Columns",
```

Comment:
not related to this line but I see a layout issue in one of the landing pages (top padding missing :wink: )
![image](https://github.com/user-attachments/assets/f531d0f4-d519-48d5-b9b1-079048cadb8b)